### PR TITLE
Image tags

### PR DESCRIPTION
**Pull request by laeschjs**
Closes #X

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ AllCops:
     - 'node_modules/**/*'
     - 'bin/*'
     - 'config/initializers/simple_form.rb'
+    - 'db/migrate/*'
```
Comment (bhargav265) [inline .rubocop.yml]: I dont think we have to add the migration files to the rubocop.yml file

File: app/views/images/new.html.erb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <%= simple_form_for @image do |f| %>
   <%= f.input :url, label: 'Enter the Image URL:'%>
+  <%= f.input :tag_list, label: 'Image Tag:' %>
```
Comment (bhargav265) [inline app/views/images/new.html.erb]: Lets write a test for this

File: app/views/images/show.html.erb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <%= flash[:notice] %>
 URL: <%= @image.url %>
 <br />
+Tags: <%= @image.tag_list %>
```
Comment (bhargav265) [inline app/views/images/show.html.erb]: Lets write a test for this too

File: app/controllers/images_controller.rb [hunk 4]
```diff
@@ -23,6 +23,6 @@ def show
   private
 
   def create_params
-    params.require(:image).permit(:url)
+    params.require(:image).permit(:url, :tag_list)
```
Comment (bhargav265) [inline app/controllers/images_controller.rb]: Lets write a test for this too

File: .rubocop.yml (hunk 1 above)
Comment (laeschjs) [reply .rubocop.yml]: If I don't then it throws like 20 errors

File: app/views/images/new.html.erb (hunk 2 above)
Comment (laeschjs) [reply app/views/images/new.html.erb]: fixed

File: app/views/images/show.html.erb (hunk 3 above)
Comment (laeschjs) [reply app/views/images/show.html.erb]: fixed

File: app/controllers/images_controller.rb (hunk 4 above)
Comment (laeschjs) [reply app/controllers/images_controller.rb]: fixed

File: test/controllers/images_controller_test.rb [hunk 5]
```diff
@@ -26,11 +29,26 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
...
 
+  test 'should create with tags' do
+    assert_difference 'Image.count' do
+      post images_path, params: { image:
+                                    {
+                                      url: 'https://partycity6.scene7.com/is/image/PartyCity/_pdp_sq_?$_1000x1000_$&$product=PartyCity/P590860',
+                                      tag_list: 'test'
+                                    } }
+    end
+    assert_redirected_to image_path(Image.last)
```
Comment (bhargav265) [inline test/controllers/images_controller_test.rb]: I think it is a good idea to test the flash message here.

Comment (laeschjs) [reply test/controllers/images_controller_test.rb]: fixed

**Pull request merged**